geode-core/src/main/java/org/apache/geode/distributed/internal/ClusterDistributionManager.java

```diff
@@ -790,8 +790,6 @@ private ClusterDistributionManager(RemoteTransportConfig transport,
 
       localAddress = membershipManager.getLocalMember();
 
-      membershipManager.postConnect();
```

Comment:
What was this call to postConnect removed?

Reply:
That was accidental - I've reinstated the postConnect().  I removed it when I was doing some other work and forgot to revert it in my initial commit.

Reply:
Did it not fail some test to leave this out?

Reply:
I did that when exploring membership components & forgot to revert it for this commit.  No, it made no tests fail because the only implementation of that method is empty.